### PR TITLE
Feature: Persian language

### PR DESCRIPTION
**Pull request by CodeWithEmad**
Hello everyone
I'm glad to announce that we translated all openedx components in transifex to Persian and fa_IR is 100% translated and 100% reviewed.

in this PR, I've added:
new translations to `i18n/messages/fa_IR.json`,
`fa-ir` in messages in `src/i18n/index.js`,
`fa-ir` code in `constants.js` in `src/account-settings/site-language`,
and `fa_IR` code in `transifex_langs` variable in `Makefile`.

is there anything else I should do?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-footer": "10.3.0",
     "@edx/frontend-component-header": "2.6.1",
-    "@edx/frontend-platform": "1.15.6",
+    "@edx/frontend-platform": "2.6.2",
```
Comment (hurtstotouchfire) [inline package.json:35]: We're actually [on 2.6.2 now](https://github.com/openedx/frontend-app-profile/blob/master/package.json#L35), I'm not sure why this diff still shows.

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad! Just seeing if this is still being pursued? If so, branch conflicts will need to be resolved.

CC: @hurtstotouchfire

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad! Friendly ping on resolving the branch conflicts :)

**Comment by CodeWithEmad** [comment]
> Hi @CodeWithEmad! Friendly ping on resolving the branch conflicts :)

Hi @mphilbrick211. totally forgot about this MR. 
conflicts resolved.

**Comment by mphilbrick211** [comment]
Thanks, @CodeWithEmad! I'll ask my team to authorize tests to run on this PR.

**Comment by e0d** [comment]
@mphilbrick211 tests approved.

**Comment by mphilbrick211** [comment]
@hurtstotouchfire looks like this is ready for review!

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad - my apologies, but could you please resolve the branch conflicts that have popped up here? I'm looking into getting this reviewed for you.

**Comment by jsnwesson** [comment]
While this branch's conflict gets resolved, I'll be bringing this in for review in my team's kanban board! 😃

**Review by jsnwesson — APPROVED** [review]

**Comment by jsnwesson** [comment]
@CodeWithEmad everything looks good! Just going to wait on approving until this branch conflict gets resolved.

**Comment by CodeWithEmad** [comment]
sorry for the late response. I was away because of a medical condition.
I'll resolve the conflicts asap.

**Review by jsnwesson — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@CodeWithEmad 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-profile&entry.867055334=https://github.com/openedx/frontend-app-profile/pull/620&entry.1484655318=https://github.com/CodeWithEmad&entry.752974735=2022-10-17+06:24&entry.1917517419=2023-07-12+17:07&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by jsnwesson** [comment]
Thanks for fixing the conflict! Hope you're feeling better!

**Pull request merged**